recipe/meta.yaml

```diff
@@ -55,10 +54,9 @@ outputs:
     build:
       string: {{ build_type }}_py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }}   #[build_type == 'cpu']
       string: {{ build_type }}{{ cudatoolkit | replace(".*", "") }}_py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }} #[build_type == 'cuda']
-{% if build_type == 'cuda' %}
       script_env:
-        - CUDA_HOME
-{% endif %}
+        - MKL_FEATURE #[x86_64]
```

Comment:
Will this still work for Power cpu only builds? In that case `script_env` will be empty.

Reply:
We don't qualify CUDA_HOME with #[build_type == 'cuda'] anywhere else, but perhaps we should. Assuming the conda render doesn't barf when given an empty category.

Reply:
I did build on Power for build_type cpu and the recipe didn't fail. I'll still check it once again.

Reply:
I rechecked this and found that it still works if qualifiers make script_env empty for some configuration. But we anyway don't need this change as we have changed the approach of implementation.